### DISCUSSION TITLE
inverse search does not work

### DISCUSSION THREAD

**Discussion by nimagitac** [sumatrapdfreader/sumatrapdf#3113] [Documentation]
Hello everybody,
There is a problem that prevents Sumatra to do the "inverse search" with TEXnicCenter. I have done a search and checked all of the possible solutions, but non worked. My setting in TeXnic is:
**Latex**
Compiler:
  path:
C:\Users\myname\AppData\Local\Programs\MiKTeX\miktex\bin\x64\pdflatex.exe
-synctex=-1 -interaction=nonstopmode "%Wm"

**Viewer:**
 Executable path:
"C:\Users\myname\AppData\Local\SumatraPDF\SumatraPDF.exe" -inverse-search "\"C:\Program Files\TeXnicCenter\TEXCNTR.EXE\" /ddecmd \"[goto(’%f’,’%l’)]\""

project output:
Commandline
"%bm.pdf"

Forward search:
DDE
[ForwardSearch("%bm.pdf","%Wc",%l,0,0,1)]
Sumatra
Control

What is the solution?

Thanks

**Comment by GitHubRulesOK** [comment]
try --synctex=-1 (note the doubled - prefix) and see if it still builds a synctex file
you should not need ` -inverse-search ""C:\Program Files\TeXnicCenter\TEXCNTR.EXE" /ddecmd "[goto(’%f’,’%l’)]""` since it should be correct in the settings file after first call

so does forward search correctly open compiled file at correct page (test it goes to 2nd when required)
however, there is a known problem with 64bit recent version and compressed synctex so check the synctex that's built when pdf is open does it have same day-time as pdf and is it NOT .synctex.gz (if it has the .gz extension it is potential issue with recent SumatraPDF 64bit)
